pw.mmk.OpenFreebuds.yml

```diff
@@ -0,0 +1,59 @@
+id: pw.mmk.OpenFreebuds
+command: openfreebuds_qt
+
+sdk: org.kde.Sdk
+runtime: org.kde.Platform
+base: com.riverbankcomputing.PyQt.BaseApp
+runtime-version: '6.7'
+base-version: '6.7'
+
+finish-args:
+  # Connect to Bluetooth devices through RFCOMM / SPP protocol
+  - --allow=bluetooth
+  - --share=network
```

Comment:
why is network required ? `allow=bluetooth` should be sufficient I think ?

Reply:
> `allow=bluetooth` should be sufficient I think ?

But it wasn't... Idk why, but Python didn't want establish a Bluetooth socket connection without network permission. Release build didn't have enough logging level to show full exception, but as I remember it reports something like `ConnectionResetError`.

![Screenshot_20240928_211832](https://github.com/user-attachments/assets/767c9894-864b-4f03-ba57-84679d3476f0)

When I enable network access back, it works:

![Screenshot_20240928_212116](https://github.com/user-attachments/assets/9749bb34-13e1-47ac-b529-e9b41a8f50fa)

Reply:
what library are you using ?

Reply:
For Bluetooth, nothing, Python's built-in `socket` module supports establishing connections through Bluetooth RFCOMM / SPP.

Reply:
Built a variant with extended logging to get an exception, and here it is, it literally can't connect without network access...

![Screenshot_20240928_212815](https://github.com/user-attachments/assets/1383eec2-65de-4f82-ae15-cd4875c12a13)

Reply:
socket access need network permission even AF_UNIX sockets. It's quite unfortunate.

Reply:
Looks like it's so....

Reply:
Yea